testnets/alloratestnet/chain.json

```diff
@@ -0,0 +1,174 @@
+        "address": "testnet-seeds.polkachu.com:26756"
+      },
+      {
+        "id": "2f73fac757fc72d937062f52e8cd4c50f9f44e30",
+        "address": "seeds.bh.rocks:59156"
+      },
+      {
+        "id": "284313184f63d9f06b218a67a0e2de126b64258d",
+        "address": "seeds.silknodes.io:15031"
+      }
+    ],
+    "persistent_peers": [
+      {
+        "id": "a63107a2244cdd2e5a1328e6edff130abaaee1d4",
+        "address": "15.204.101.28:26656"
+      },
+      {
+        "id": "2339ddf13f6d8ae2c78a0d0bce8001a19ea2f38c",
+        "address": "15.204.101.34:26656"
+      },
+      {
+        "id": "e158a85ca8fbcef5813736f8ef73e308a0ad2b3a",
+        "address": "15.204.101.35:26656"
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "https://allora-rpc.testnet.allora.network",
+        "provider": "allora"
+      },
+      {
+        "address": "https://rpc.ankr.com/allora_testnet",
+        "provider": "ankr"
+      },
+      {
+        "address": "https://allora-testnet-rpc.polkachu.com",
+        "provider": "polkachu"
+      },
+      {
+        "address": "https://allora-rpc.sh.moonlet.cloud/public",
+        "provider": "moonlet"
+      }
+    ],
+    "rest": [
+      {
+        "address": "https://allora-api.testnet.allora.network",
+        "provider": "allora"
+      },
+      {
+        "address": "https://allora-testnet-api.polkachu.com",
+        "provider": "polkachu"
+      },
+      {
+        "address": "https://testnet-allora-api.lavenderfive.com",
+        "provider": "lavenderfive"
+      },
+      {
+        "address": "https://rpc.ankr.com/http/allora_testnet",
+        "provider": "ankr"
+      }
+    ],
+    "grpc": [
+      {
+        "address": "allora-grpc.testnet.allora.network:443",
+        "provider": "allora"
+      },
+      {
+        "address": "allora-testnet-grpc.polkachu.com:26790",
+        "provider": "polkachu"
+      }
+    ]
+  },
+  "explorers": [
+    {
+      "kind": "ping.pub",
+      "url": "https://explorer.testnet.allora.network",
+      "tx_page": "https://explorer.testnet.allora.network/allora-testnet-1/tx/${txHash}",
+      "account_page": "https://explorer.testnet.allora.network/allora-testnet-1/account/${accountAddress}"
+    },
+    {
+      "kind": "Allora Studio",
+      "url": "https://explorer.testnet-1.allora.network",
+      "tx_page": "https://explorer.testnet-1.allora.network/explorer/transactions/${txHash}",
+      "account_page": "https://explorer.testnet-1.allora.network/profile/${accountAddress}"
+    },
+    {
+      "kind": "Moonlet",
+      "url": "https://explorer.moonlet.cloud/allora",
+      "tx_page": "https://explorer.moonlet.cloud/allora/tx/${txHash}",
+      "account_page": "https://explorer.moonlet.cloud/allora/account/${accountAddress}"
+    }
+  ],
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/alloratestnet/images/allora.png",
```

Comment:
Note the the padding on the images will make these logos appear very small in most applications. A version without padding should be used instead if it's intended to be as big as possible.